src/caffe/solver.cpp

```diff
@@ -434,7 +434,8 @@ Dtype SGDSolver<Dtype>::GetLearningRate() {
         (Dtype(1.) + exp(-this->param_.gamma() * (Dtype(this->iter_) -
           Dtype(this->param_.stepsize())))));
   } else {
-    LOG(FATAL) << "Unknown learning rate policy: " << lr_policy;
+    rate = Dtype(0.);
```

Comment:
One possible idea: keep the learning rate schedule, and treat it as a multiplier on the AdaDelta update step size.  The only ugly part of this solution is that it would require the user to specify `base_lr: 1 lr_policy: 'fixed'` in order to get the default behavior.